WordPress/Classes/Models/UserProfile.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+open class UserProfile {
```

Comment:
If these are truly immutable values I'd suggest making this a struct and all of the properties constants. You then get a default initializer for free!